submit-a-pull-request/BirkhoffLee/hello-world.md

```diff
@@ -0,0 +1,2 @@
+Hello world
+I am BirkhoffLee. :tada:
```

Comment:
There should be an empty line (new line, with no space characters) at the end of the file.